lib/Common/DataStructures/FixedBitVector.h

```diff
@@ -317,7 +318,7 @@ void BVFixed::SetRange(Container* value, BVIndex start, BVIndex len)
     {
         // Calculation above might overflow the original container.
         // normalize the overflow value. LE only
-        temp = (*((char*)bits)) + (*(((char*)bits) + 1));
+        //temp = (*((char*)bits)) + (*(((char*)bits) + 1));
```

Comment:
Changing back to the previous version, since I'm just marking this function as NO_SANITIZE_ADDRESS anyway.